### PR TITLE
RTL arrow problems

### PR DESCRIPTION
**Pull request by yosilevy**
Updated style check since load sequence must have changed and computed style is only right after full page load

### CODE REVIEW

File: src/components/ha-icon-next.ts [hunk 1]
```diff
@@ -6,7 +6,8 @@ import { HaIcon } from "./ha-icon";
 export class HaIconNext extends HaIcon {
   public connectedCallback() {
     this.icon =
-      window.getComputedStyle(this).direction === "ltr"
+      (document.querySelector("home-assistant")! as HTMLElement).style
```
Comment (balloob) [inline src/components/ha-icon-next.ts]: We cannot assume that this element is always available, like in Home Assistant Cast

File: src/components/ha-icon-prev.ts [hunk 2]
```diff
@@ -5,13 +5,14 @@ import { HaIcon } from "./ha-icon";
...
-
-    // calling super after setting icon to have it consistently show the icon (otherwise not always shown)
     super.connectedCallback();
+
+    setTimeout(() => {
+      this.icon =
+        window.getComputedStyle(this).direction === "ltr"
+          ? "hass:chevron-left"
+          : "hass:chevron-right";
+    }, 100);
```
Comment (balloob) [inline src/components/ha-icon-prev.ts]: Please add a comment to each setTimeout.

Comment (balloob) [reply src/components/ha-icon-prev.ts]: did you try moving this to `firstUpdated` instead of connectedCallback ?

Comment (yosilevy) [reply src/components/ha-icon-prev.ts]: Currently HA-ICON is an HtmlElement

Comment (balloob) [reply src/components/ha-icon-prev.ts]: `ha-icon` extends `iron-icon`, which is a Polymer element, so you should be able to override `ready()`

**Review by balloob — APPROVED** [review]

**Pull request merged**